src/services/.oidc/serverless.yml

```diff
@@ -11,7 +11,6 @@ plugins:
 
 provider:
   name: aws
-  runtime: nodejs16.x
```

Comment:
I don't think we want to remove this and accept a default as I'm not sure that default will always be what we want.